RadixWallet/Features/AccountRecoveryScan/Coordinator/AccountRecoveryScanCoordinator.swift

```diff
@@ -145,7 +145,7 @@ public struct AccountRecoveryScanCoordinator: Sendable, FeatureReducer {
 		active: IdentifiedArrayOf<Account>,
 		inactive: IdentifiedArrayOf<Account>
 	) -> Effect<Action> {
-		let sortedAccounts: IdentifiedArrayOf<Account> = { () -> IdentifiedArrayOf<Account> in
+		let sortedAccounts: Accounts = { () -> IdentifiedArrayOf<Account> in
```

Comment:
this is the same thing! `Accounts` **is** (a typealias of) ` IdentifiedArrayOf<Account>`.....